runner.ts

```diff
@@ -306,23 +307,30 @@ export default class Runner {
 
   calculateAggregatedValues(
     prices: PriceDataBeforeAggregation[]): PriceDataAfterAggregation[] {
-    return prices.map((p) =>
-      aggregators[this.manifest.priceAggregator].getAggregatedValue(p));
+    const aggregatedPrices: PriceDataAfterAggregation[] = [];
+    const aggregator = aggregators[this.manifest.priceAggregator];
+    for (const price of prices) {
+      try {
+        const priceAfterAggregation = aggregator.getAggregatedValue(price);
+        aggregatedPrices.push(priceAfterAggregation);
+      } catch (e) {
+        logger.error(e.stack);
```

Comment:
I'm wondering what should we put if there is an error in aggregation, undefined?